README.md

```diff
@@ -61,8 +61,8 @@
 ### Key Features <a name="key-features"></a>
 
 
-- **[SPA]()**
-- **[Calculator]()**
+- **[REACT APP]()**
+- **[BOOK STORE]()**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [optional] you can remove the live demo section from your README for now, since your project is yet to be deployed. You can add it later on
- [optional] please the FAQ is optional but should have some content if you decide to use it. Kindly add information to it, or you can remove the section as well . This will make your README file more professional. Nice work.